### ISSUE TITLE
MA0028 should not be reported for StringBuilder.Insert(....)

### ISSUE THREAD

**Issue by drieseng** [meziantou/Meziantou.Analyzer#372]
**Context**
Rule identifier: _MA0028_
Target Framework: _.NET 4.8_
Language version: _10.0_

**Describe the bug**
MA0028 is reported for a call to `StringBuilder.Insert(int index, string value)` where `value` is `string.Format(...)`.

**To Reproduce**
````c#
var sb = new StringBuilder();
_ = sb.Append('X');
_ = sb.Insert(0, string.Format(CultureInfo.InvariantCulture, "{0}{1}", "A", "B"));
````

**Additional context**
As expected, the fixer does nothing.

PS. I'm introducing analyzers in a legacy code base, so I'm running into code that I wouldn't write myself.

**Comment by meziantou** [comment]
Should be fixed by aa2c8f1f6e9e837163fd45366515fd590d225c7f (version 1.0.723)
